spec/tasks/themes_spec.rb

```diff
@@ -118,4 +118,158 @@
       expect(theme.remote_theme.local_version).to eq(original_local_version)
     end
   end
+
+  describe "themes:deduplicate_horizon" do
+    fab!(:user)
+    fab!(:remote_theme) do
+      RemoteTheme.create!(
+        remote_url: "https://github.com/discourse-org/horizon.git",
+        minimum_discourse_version: "2.5.0",
+      )
+    end
+    fab!(:manual_horizon_theme) do
```

Comment:
```suggestion
    fab!(:remote_horizon_theme) do
```

I think this is a bit clearer